Doc/using/windows.rst

```diff
@@ -23,8 +23,9 @@ available for application-local distributions.
 
 As specified in :pep:`11`, a Python release only supports a Windows platform
 while Microsoft considers the platform under extended support. This means that
-Python |version| supports Windows 8.1 and newer. If you require Windows 7
-support, please install Python 3.8.
+Python |version| supports Windows 10 and newer. If you require Windows 7
+support, please install Python 3.8. If you require Windows 8.1 support,
+please install Python 3.12.
```

Comment:
```suggestion
please install Python 3.13.
```

Reply:
Officially, 3.13 doesn't support 8.1. There's a difference between stated support and what will actually work and is blocked/not-blocked by the installer.

If something comes up in 3.13 where the fix would prevent working on 8.1, we can do it. Or we can choose not to do it. But we aren't painted into a corner by over-specifying anything.

[PEP 11](https://peps.python.org/pep-0011/) is the rule, not this particular piece of documentation.

Reply:
oh, I understand. Thank you answer!